geospaas/base_viewer/templates/base_viewer/elements.html

```diff
@@ -15,7 +15,7 @@
         <td class="place_ds">
             <div class="geometry_ref" ajax_url="{% url 'base_viewer:geometry_geojson' ds.geographic_location.pk %}">
             {% for url in ds.dataseturi_set.all %}
-            {{ url.uri}} <br>
+            <a href="{{url.uri}}">Start: {{url.dataset.time_coverage_start}} {{url.dataset.entry_id}}</a>  <br>
```

Comment:
We might want to display the data about the dataset in multiple columns rather than everything in one.

We probably also want to adjust which data is displayed about the dataset. For example, users might not be interested in the `entry_id` much.